src/components/ContributionsEpic.mocks.ts

```diff
@@ -0,0 +1,28 @@
+const content = {
```

Comment:
Minor, but could we call this file something else? `Mock` often means mock objects, whereas this is closer to test data really. Maybe a `testData` folder or something?

Reply:
Fair enough, I've updated the file name and exports to `.testData`, looks slightly on in the project tree but it's descriptive and accurate. But I've kept these files together with the component and story files... I think that's better than a `testData` folder as it keeps everything related to the same component together.